sources/packages/api/src/route-controllers/program-info-request/program-info-request.controller.ts

```diff
@@ -240,6 +243,32 @@ export class ProgramInfoRequestController {
       );
     }
   }
+  /**
+   * Get all active application of a location in a institution
```

Comment:
Please correct the grammar to: "Get all active applications of a location in an institution
with application_status is completed."